src/minimax_server/core.clj

```diff
@@ -1 +1,10 @@
-(ns minimax-server.core)
+(ns minimax-server.core
+  (:import [scarvill.httpserver.server Server])
+  (:require [minimax-server.computer-player :as computer]
+            [minimax-server.server-configuration :refer [minimax-server-config]]))
+
+(def ttt-server
+  (new Server (minimax-server-config 5000 computer/stupid-move)))
```

Comment:
`computer/stupid-move` seems like a fairly low level detail to reference in `main`.

What happens if/when there are multiple endpoints? The current design seems like it would encourage changes here.